### PR TITLE
Add CustomArrayList and CustomHashSet classes

### PR DESCRIPTION
**Pull request by kolosokvit**

### CODE REVIEW

File: src/main/java/storages/CustomHashSet.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package storages;
+
+import java.util.HashMap;
+import java.util.Iterator;
+
+public class CustomHashSet<E> {
+    private static final Object PRESENT = new Object();
+    private HashMap<E, Object> elements;
```
Comment (AtticSiol) [inline src/main/java/storages/CustomHashSet.java]: do not use HashMap, instead use array, ArrayList ot LinkedList

File: src/main/java/storages/CustomArrayList.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
+package storages;
```
Comment (AtticSiol) [inline src/main/java/storages/CustomArrayList.java:1]: add examples of how to use each method of the storages into your main method

Comment (kolosokvit) [reply src/main/java/storages/CustomArrayList.java:1]: Fixed in https://github.com/kolosokvit/ticket-service/pull/3/commits/5d4f5d14b38c811869c54cba7df89aa28f97c409

File: src/main/java/storages/CustomHashSet.java (hunk 1 above)
Comment (kolosokvit) [reply src/main/java/storages/CustomHashSet.java]: Fixed in https://github.com/kolosokvit/ticket-service/pull/3/commits/091895e1f96fc30220d1f6eb22b73efc0992bf45

**Review by AtticSiol — APPROVED** [review]

**Pull request merged**